### PR TITLE
Added MobiMart logo icon in title bar of website

### LINKED ISSUE #44: Add MobiMart logo in title bar
Coding

### PR DESCRIPTION
**Pull request by QAZIMAAZARSHAD**
#### Issue Number
ISSUE #44 

Added logo icon in title bar

#### Checklist

- [✔ ] My code follows the code style of this project.
- [✔ ] I have performed a self-review of my own code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [✔ ] I have commented my code, particularly in hard-to-understand areas.
- [ ✔] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ✔] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: MobiMart/templates/base.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<!-- Title bar icon -->
+    <link rel = "icon" href ="./static/images/Logo.png" "type "image/x-icon">
```
Comment (kavania2002) [inline MobiMart/templates/base.html]: shouldn't these tag be added in title section?

**Review by kavania2002 — COMMENTED** [review]
You don't need to add icon to every page, as the index.html, cart.html, checkout.html are extended from base.html so no need. It was right that you need in register.html and login.html

File: MobiMart/templates/base.html (hunk 1 above)
Comment (QAZIMAAZARSHAD) [reply MobiMart/templates/base.html]: Yes, I added these tags in the title section first but when I opened base.html in chrome icon was not updated.
So I added these tags above {% load static %}.

Now, title icon is updated.

Comment (QAZIMAAZARSHAD) [reply MobiMart/templates/base.html]: Yes, you were right.

So, should I create a new pull request correcting all these errors and close this one ?

**Comment by kavania2002** [comment]
Yep it's fine now.
Good job.

**Pull request merged**